projects/1.18/assets/backpacked/backpacked/lang/zh_cn.json

```
@@ -1,7 +1,83 @@
 {
-  "curios.identifier.backpacked": "可塞入背包",
-  "backpacked.configgui.keepBackpackOnDeath": "在死亡时保留背包",
-  "container.backpack": "背包",
+  "item.backpacked.backpack": "背包",
+  "block.backpacked.oak_backpack_shelf": "橡木背包架",
+  "block.backpacked.spruce_backpack_shelf": "云杉木背包架",
+  "block.backpacked.birch_backpack_shelf": "白桦木背包架",
+  "block.backpacked.jungle_backpack_shelf": "丛林木背包架",
+  "block.backpacked.dark_oak_backpack_shelf": "深色橡木背包架",
+  "block.backpacked.acacia_backpack_shelf": "金合欢木背包架",
+  "block.backpacked.crimson_backpack_shelf": "绯红菌背包架",
+  "block.backpacked.warped_backpack_shelf": "诡异菌背包架",
   "key.backpack": "打开背包",
-  "item.backpacked.backpack": "背包"
-}
\ No newline at end of file
+  "container.backpack": "背包",
+  "backpacked.configgui.keepBackpackOnDeath": "死亡时保留背包",
+  "backpacked.configgui.backpackInventorySizeColumns": "背包物品栏列数",
+  "backpacked.configgui.backpackInventorySizeRows": "背包物品栏行数",
+  "curios.identifier.backpacked": "Backpacked",
+  "button.categories.backpacked": "Backpacked",
+  "backpacked.gui.locked": "锁定",
+  "backpacked.toast.unlocked_backpack": "已解锁背包",
+  "backpacked.button.open_backpack": "打开背包",
+  "backpacked.button.reset": "重置",
+  "backpacked.button.save": "保存",
+  "backpacked.button.customise.tooltip": "自定义",
+  "backpacked.button.config.tooltip": "配置选项",
+  "backpacked.button.show_effects.tooltip": "显示效果",
+  "backpacked.button.show_with_elytra.tooltip": "同时显示鞘翅",
+  "backpacked.button.show_enchantment_glint.tooltip": "显示附魔光效",
+  "backpacked.title.customise_backpack": "自定义背包",
+  "message.backpacked.player_opened": "一位玩家打开了你的背包",
+  "backpacked.backpack.standard": "标准",
+  "backpacked.backpack.classic": "经典",
+  "backpacked.backpack.bamboo_basket": "竹篮",
+  "backpacked.backpack.bamboo_basket.unlock": "喂竹子给10只不同的熊猫来解锁这个背包",
+  "backpacked.backpack.rocket": "火箭",
+  "backpacked.backpack.rocket.unlock": "用鞘翅飞行50km来解锁这个背包",
+  "backpacked.backpack.mini_chest": "迷你箱子",
+  "backpacked.backpack.mini_chest.unlock": "找到5个未开封的埋藏宝箱来解锁这个背包",
```

```suggestion
  "backpacked.backpack.mini_chest.unlock": "找到5个未开封的埋藏的宝箱来解锁这个背包",
```
[原版词](https://minecraft.fandom.com/zh/wiki/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96)